src/plugin/plugin.ts

```diff
@@ -606,12 +636,14 @@ class Purchases {
    * @param {string?} userDefaultsSuiteName An optional string. iOS-only, will be ignored for Android. 
    * Set this if you would like the RevenueCat SDK to store its preferences in a different NSUserDefaults 
    * suite, otherwise it will use standardUserDefaults. Default is null, which will make the SDK use standardUserDefaults.
+   * @param {boolean} useAmazon Required to configure the plugin to be used in the Amazon Appstore. 
    */
   public static configure(
     apiKey: string,
     appUserID?: string | null,
     observerMode: boolean = false,
-    userDefaultsSuiteName?: string
+    userDefaultsSuiteName?: string | null,
+    useAmazon: boolean = false
```

Comment:
Should we just mark this method as deprecated instead? And if people want to use amazon they need to use `PurchaseConfiguration`.

Reply:
I like the idea of making it mandatory to use `PurchasesConfiguration` if you want to configure for Amazon